torch/linalg/__init__.py

```diff
@@ -84,6 +84,54 @@
     True
 """)
 
+cholesky_ex = _add_docstr(_linalg.linalg_cholesky_ex, r"""
```

Comment:
I like that we're documenting this completely separately from linalg.cholesky because it's an experimental function

Reply:
That said, we should probably do two things:

- notify the user this function is a PROTOTYPE that may change in a future PyTorch release
- explain how it's different from torch.linalg.cholesky (it always returns the infos, with check_errors=False the computation is the same, with check_errors=True the computation skips error checking, which can be useful if users would prefer to do the error checking themselves or they know the inputs will not throw an error)

Reply:
I added a warning, is it okay?
```
.. warning:: This function is "experimental" and it may change in a future PyTorch release.
```
Not sure on the use of quotes though.